routes/web.php

```diff
@@ -19,8 +19,13 @@
     return view('category');
 });
 
+// Route::get('/user', function () {
+//     return view('user');
+// });
+
 Auth::routes();
 
 Route::get('/home', 'HomeController@index')->name('home');
 Route::get('/users', 'UserController@users');
+Route::get('/user/{id}/profile', 'UserController@show')->name('profile');
 // Route::get('/users', 'UserController@users');
```

Comment:
if you also do not need this, please remove

Reply:
It's needed to show profile page.

Reply:
I meant the one below the profile, sorry, I pointed at a wrong line.